### DISCUSSION TITLE
When we will get laravel-livewire-wizard new version which compatible with livewire 3?

### DISCUSSION THREAD

**Discussion by in-chetan-thumar** [spatie/laravel-livewire-wizard#91] [General]
Hello Spatie Team,

When we will get laravel-livewire-wizard new version which compatible with Livewire 3?

Regards,
Chetan

**Comment by ju5t** [comment]
When you asked this question Livewire 3 was still in beta. This may sound a bit unfriendly, but that's not the time to ask for support, also, if you had checked the open PR's you would have found one about Livewire 3 and this question wouldn't be necessary. 

If you need it to be available quickly, it always helps if you test the PR, give feedback on what works and what doesn't work, so it can be improved and released.
